examples/convert_legacy_llama.py

```diff
@@ -1021,35 +1042,32 @@ def pick_output_type(model: LazyModel, output_type_str: str | None) -> GGMLFileT
     raise ValueError(f"Unexpected combination of types: {name_to_type}")
 
 
-def model_parameter_count(model: LazyModel) -> int:
-    total_model_parameters = 0
-    for i, (name, lazy_tensor) in enumerate(model.items()):
-        sum_weights_in_tensor = 1
+def per_model_weight_count_estimation(tensors: Iterable[tuple[str, LazyTensor]]) -> tuple[int, int, int]:
```

Comment:
Shouldn't this be moved to `utility.py`?

Reply:
No this shoud stay in `convert_legacy_llama.py`, because `LazyTensor` is specific to `convert_legacy_llama.py` (it does *not* come from `gguf-py/gguf/lazy.py`).